### DISCUSSION TITLE
Dashboard -> all products: The stock column shows that there are 5 items, and inside everything is zero

### DISCUSSION THREAD

**Discussion by agsmartwolf** [swellstores/community#245] [Help] [answered]
On admin dashboard in all products section, if a product has a certain color/size (let's say 5 units) and you delete this color/size without first resetting the availability, then this quantity will be taken into account in all products and it will not be deleted in any way from there, even if the rest of the colors/sizes are zero

**Comment by agsmartwolf** [comment]
We've removed and recreated our product for production (product name was: "touring bungee leash")


But the problem is actual for all of products. You can see it for our test product below:
All products view:
![telegram-cloud-photo-size-2-5399947332428089928-y](https://github.com/swellstores/community/assets/109105178/a7b5e5eb-3165-4e2c-909d-beae2b832317)


Product details:
![telegram-cloud-photo-size-2-5399947332428089929-y](https://github.com/swellstores/community/assets/109105178/2012b546-efbc-428e-b9de-3a3ddf1f784d)

**Reply by logeshswell** [reply, marked as answer]
@agsmartwolf Thanks for bringing it to us. I confirm that it is a bug and let me create a bug ticket internally for faster resolution.

**Reply by steven-k1** [reply]
@agsmartwolf A fix has been deployed for this issue. If you experience any further issues with this, please let us know.
